node-package/bin/dukat-cli.js

```diff
@@ -23,18 +28,37 @@ function run(command, args) {
     return child;
 }
 
+function printInvalidJavaHome() {
+    printError("Your \"JAVA_HOME\" environment variable points to an invalid directory.");
+    printError("As of now \"dukat\" requires Java Runtime Environment to be installed.");
+    printError("Either set \"JAVA_HOME\" to a proper Java installation,");
+    printError("or make sure \"java\" is callable from commandline and unset \"JAVA_HOME\".");
+}
+
 function printNoJava() {
-    printError("It looks like you don't have java installed or it's just not reachable from command-line");
+    printError("\"java\" is not callable from commandline and no \"JAVA_HOME\" environment variable is set.");
     printError("As of now \"dukat\" requires Java Runtime Environment to be installed.");
+    printError("Either set \"JAVA_HOME\" to a proper Java installation,");
+    printError("or make sure \"java\" is callable from commandline.");
 }
 
-function guardJavaExists() {
-    exec("java -version", function (error, stdout, stderr) {
-        if (error) {
+function findJavaExecutable() {
+    var javaHome = process.env.JAVA_HOME
+    var result = javaHome ? path.join(javaHome, "bin", "java") : "java"
+    try {
+        execSync('"' + result + "\" -version", {
```

Comment:
I'm against of this approach so let's discuss. One thing is to rely on JAVA_HOME - that's definitely an improvement. 
The other thing is to check java -v every time we are calling dukat. 
May be there's sense to do it in some sort of debug or diagnostics mode but each time seems to me be an overkill.

Reply:
To be honest, that approach ist not by me, but by you (or whoever orginally wrote that code).
I didn't change that logic.
The only change I made is, that JAVA_HOME is considered for finding Java.
`findJavaExecutable` ist called at the exact same one place `guardJavaExists` was called before.
I just changed its name and made it return the path to be used to call Java later on.

Reply:
A-ha-ha, now I feel **very** stupid! Sorry for misinterpreting, I apologize ))
This PR looks good to me. Thank you for your contribution. I guess it's time to introduce a contributors list - will create task on this.